pages/index.js

```diff
@@ -20,7 +18,10 @@ export default function Home() {
 
   useEffect(() => {
     const fetchData = async () => {
-      // const data = await fetchCIData();
+      const response = await fetch(
+        "https://raw.githubusercontent.com/a1icja/kata-dashboard-next/refs/heads/latest-dashboard-data/data/job_stats.json"
+      );
```

Comment:
Would this be good to also fetch dynamically vs hard code?

Reply:
The closest we could get to doing this dynamically is making it an env var for when we build for deploy. Either way, we would still need to change a file in the repo, so I'm not sure if there's a benefit, but lmk.